lib/orator/index.js

```diff
@@ -327,18 +327,24 @@ class Orator {
           avatarURL
         } = response
 
-        const hook = await findOrCreateManagedWebhook(
-          msg.channel,
-          bot.user.id,
-          this.webhookName
-        )
+        try {
+          const hook = await findOrCreateManagedWebhook(
+            msg.channel,
+            bot.user.id,
+            this.webhookName
+          )
 
-        newMessage = await bot.executeWebhook(hook.id, hook.token, {
-          ...content,
-          username,
-          avatarURL,
-          wait: true
-        })
+          newMessage = await bot.executeWebhook(hook.id, hook.token, {
+            ...content,
+            username,
+            avatarURL,
+            wait: true
+          })
+        } catch (_) {
```

Comment:
I'm pretty sure you can just do `catch {`

Reply:
we still want to have the `switch` checking the error, we should throw it back if the error wasn't explicitly about permissions